test/spec/Feature/Query/QuerySpec.hs

```diff
@@ -1625,3 +1625,10 @@ spec = do
       get "/infinite_recursion?select=*" `shouldRespondWith`
         [json|{"code":"42P17","message":"infinite recursion detected in rules for relation \"infinite_recursion\"","details":null,"hint":null}|]
         { matchStatus = 500 }
+
+  context "invalid resource path" $ do
+    it "return http status 400" $
+      get "/first/second/third?select=*"
+      `shouldRespondWith`
+      [json| {"code":"PGRST125","details":"Request path: /first/second/third","hint":null,"message":"Invalid path specified in request URL"} |]
```

Comment:
Details don't add much, so just:
```suggestion
      [json| {"code":"PGRST125","details":"null,"hint":null,"message":"Invalid path specified in URL"} |]
```

It might bring some simplification in code too.